main/test/flix/LangSuite.scala

```diff
@@ -111,6 +111,12 @@ class LangSuite extends Suites(
   new FlixTest("Test.Exp.Concurrency.Select", "main/test/flix/Test.Exp.Concurrency.Select.flix")(Options.DefaultTest.copy(xallowredundancies = true)),
   new FlixTest("Test.Exp.Concurrency.Spawn", "main/test/flix/Test.Exp.Concurrency.Spawn.flix")(Options.TestWithLibrary),
 
+  //
+  // Lazy/Force.
+  //
+  new FlixTest("Test.Exp.Lazy", "main/test/flix/Test.Exp.Lazy.flix"),
+  new FlixTest("Test.Exp.Force", "main/test/flix/Test.Exp.Force.flix"),
+
```

Comment:
This file is kept alphabetically sorted, so please move Lazy and Force to the appropriate places :) (Yes, they will then be at separate places).